tests/server/index.js

```diff
@@ -237,7 +237,6 @@ const JOB_ID = uuid4();
       cmd += `--name ${JOB_ID} ` +
         `-e TASK_DESCRIPTION="${taskURL}" ` +
         `-e API_TOKEN="xxxxxxxxxxxxxxxx" ` +
-        `-e OS="${process.platform}" ` +
```

Comment:
This may require some work in the test section as it doesn't have the url conversion there.

Reply:
*will* require work. I believe this PR as is breaks the local test server for Mac users